.travis.yml

```diff
@@ -59,3 +54,15 @@ script:
 after_success:
 - if [ $RUN_BACKEND_TESTS == 'true' ] && [ $REPORT_BACKEND_COVERAGE == 'true' ]; then codecov; fi
 - if [ $RUN_FRONTEND_TESTS == 'true' ]; then codecov --file ../karma_coverage_reports/coverage-final.json; fi
+
+cache:
+  # Cache Oppia's dependencies.
+  directories:
+    - ../node_modules/
+    - ../oppia_tools/
+    - third_party/
+    - core/templates/prod/
```

Comment:
The first three won't change, but this one will, right? I mean, the core/templates/prod files are usually generated from dev during setup...